paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/PlaygroundTestDriver.kt

```diff
@@ -806,6 +806,9 @@ internal class PlaygroundTestDriver(
     }
 
     private fun doUSBankAccountAuthorization() {
+        Espresso.onIdle()
+        composeTestRule.waitForIdle()
```

Comment:
Was this needed as part of this work or does it help mitigate some of flakiness of `TestUSBankAccount`? If the later, might be better to just merge it outside of this PR.

Reply:
Happy to pull out into a separate PR (it would work, and be helpful outside of this PR). But it was always failing with my changes, which is why I included it in this PR.

Reply:
Ah okay, makes sense to keep it in this PR then!